README.md

```diff
@@ -1,92 +1,199 @@
+[![Build Status](https://dev.azure.com/daal/DAAL/_apis/build/status/intel.daal?branchName=master)](https://dev.azure.com/daal/DAAL/_build/latest?definitionId=3&branchName=master) ![License](https://img.shields.io/github/license/intel/daal.svg)
 
+ Intel&reg; oneAPI Data Analytics Library (oneDAL) is a powerful machine liearning library that helps speed up big data analysis.
+ Intel&reg; oneDAL solvers are also used in [Intel Distribution for Python]([https://software.intel.com/en-us/distribution-for-python](https://software.intel.com/en-us/distribution-for-python)) in scikit-learn optimization.
 
-Intel(R) oneAPI Data Analytics Library (oneDAL) is a library that helps speed up big data analysis. 
-We provide highly optimized algorithmic building blocks for all stages of data analytics: **preprocessing**, **transformation**, **analysis**, **modeling**, **validation**, and **decision making**. Our algorithms suppost **batch**, **online**, and **distributed** processing modes of computation. 
+## Build yours high-performance data science application with intel&reg; DAAL
+
+ intel&reg; DAAL use all capabilities of your hardware, which allows you to get an incredible performance boost on the classic machine learning algorithms.
+We provide highly optimized algorithmic building blocks for all stages of data analytics: **preprocessing**, **transformation**, **analysis**, **modeling**, **validation**, and **decision making**.
 
 The current version of oneDAL provides Data Parallel C++ (DPC++) API extensions to the traditional C++ interface.
 
-## Table of Contents <!-- omit in toc -->
+Check out our [examples](#examples)  and [documentation](#documentation)  for information about our API
 
-- [Technical Preview Features](#preview-features)
-- [oneDAL and Intel(R) DAAL](#onedal-and-intelr-daal)
-- [Installation](#installation)
-- [Examples](#examples)
-- [Documentation](#documentation)
-- [API](#api)
+## Improve performance of your ML application
 
+DAAL is the perfect solution if you are interested in a high-performance machine learning framework. DAAL uses machine learning algorithms optimizations and full power of your hardware to achieve greater performance than alternative solutions offers.
 
-## Technical Preview Features 
- 
-Technical preview features are introduced to gain early feedback from developers. A preview feature is subject to change in the future releases. Using a preview feature in a production code base is therefore strongly discouraged.
-The only preview feature at the moment is `MultiNodeBatch` for K-Means, a stepless distributed algorithm based on oneCCL.
+<img style="display:inline;" height=auto width=auto src="https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/IDP%20scikit-learn%20accelearation%20compared%20with%20stock%20scikit-learn.png"></a>
```

Comment:
Guess it should be a way to refer images relatively so it will not be required to specify repo name

```suggestion
<img style="display:inline;" height=auto width=auto src=".github/charts/IDP%20scikit-learn%20accelearation%20compared%20with%20stock%20scikit-learn.png"></a>
```

Reply:
Made all links relative